### DISCUSSION TITLE
Ethereum rpc support

### DISCUSSION THREAD

**Discussion by ancjf** [filecoin-project/lotus#9163] [Build and Sync] [answered]
Could Ethereum RPC be supported on Filecoin to make it easier to use Matemask?

**Comment by rjan90** [comment, marked as answer]
Hey @ancjf! Supporting Ethereum RPC is on the agenda as far as I know, but it will be an incremental delivery in itself if I recall correctly. You can read more about the Filecoin Virtual Machine milestone 2.1 here: https://github.com/filecoin-project/ref-fvm/issues/691 and the incremental delivery plan: https://github.com/filecoin-project/ref-fvm/issues/692#issue-1331791389 😄

**Reply by ancjf** [reply]
Thanks for you reply
